File: SPCode/CS/Docking/Extender/Override/DockPaneStripOverride.cs
using System.Text.RegularExpressions;
using System.Diagnostics;
using System;
using System.Drawing.Drawing2D;
using System.Windows.Forms;
using System.Collections;
using System.Drawing;
using Microsoft.VisualBasic;
using System.Data.SqlClient;
using System.Data;
using System.Collections.Generic;
using WeifenLuo.WinFormsUI;
using Microsoft.Win32;
using WeifenLuo;

namespace SoftLogik.Win
{
	namespace UI
	{
		namespace Docking
		{
			
			public class DockPaneStripOverride : DockPaneStripVS2003
			{
				
				protected internal DockPaneStripOverride(DockPane pane) : base(pane)
				{
					BackColor = SystemColors.ControlLight;
				}
			}
		}
	}
	
}

File: SPCode/Docking/Extender/Override/DockPaneStripOverride.vb
Imports System
Imports WeifenLuo.WinFormsUI
Imports System.Drawing
Imports System.Drawing.Drawing2D
Namespace UI.Docking
    Public Class DockPaneStripOverride
        Inherits DockPaneStripVS2003
        Protected Friend Sub New(ByVal pane As DockPane)
            MyBase.New(pane)
            BackColor = SystemColors.ControlLight
        End Sub
    End Class
End Namespace
